src/templates/career-details.tsx

```diff
@@ -69,6 +69,7 @@ export const PersonioHtml = styled.div`
 interface CareerPageProps {
   pageContext: {
     position: PersonioJobPosition;
+    socialCardImage: string;
```

Comment:
image comes through the context as we can't query for personio nodes right now that we could attach a field.